presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcClient.java

```diff
@@ -110,4 +117,43 @@ public void testMetadataWithFloatAndDoubleCol()
                 new JdbcColumnHandle(CONNECTOR_ID, "COL3", JDBC_DOUBLE, DOUBLE),
                 new JdbcColumnHandle(CONNECTOR_ID, "COL4", JDBC_REAL, REAL)));
     }
+
+    @Test
+    public void testCreateTable()
+    {
+        String tableName = randomUUID().toString().toUpperCase(Locale.getDefault());
```

Comment:
Use `toUpperCase(ENGLISH)` as we always want the expected behavior for hex digits.